app/src/main/res/values/strings.xml

```diff
@@ -15,6 +15,11 @@
     <string name="week_at_a_glance_meal_deleted">Removed meal</string>
     <string name="week_at_a_glance_undo_delete">Undo</string>
     <string name="week_at_a_glance_delete_menu">Delete</string>
+    <string name="week_at_a_glance_edit_servings_menu">Edit Servings&#8230;</string>
+    <string name="week_at_a_glance_edit_servings_title">Edit servings for "%1$s"</string>
```

Comment:
It looks like the quotes need to be escaped. I think you can just put `\`'s before them, or if you want to be fancy, you can use the curly double quotes. Then this would become `&#8220;%1$s&#8221;` because HTML codes.